tests/testthat/test-install.R

```diff
@@ -275,21 +274,12 @@ test_that("install() installs inferred dependencies", {
   skip_on_cran()
   renv_tests_scope("breakfast")
 
-  # use dummy library path
-  templib <- renv_scope_tempfile("renv-library-")
-  ensure_directory(templib)
-  renv_scope_libpaths(templib)
-
   # try installing packages
   install()
 
   # validate that we've installed breakfast + deps
   expect_true(renv_package_installed("breakfast"))
 
-  # try calling install once more; nothing should happen
-  records <- install()
```

Comment:
`install()` now returns one record, but my understanding is that's correct because `install()` should always re-install.

Reply:
Normally `renv::install()` (called without arguments) should be a no-op if all installed packages are up-to-date, e.g.

```
> renv::install()
Downloading 24 package(s) and their dependencies
Done

* There are no packages to install.
```

Reply:
I think either you or I was confused by our slack thread, because I thought that this was what you wanted.